src/Controllers/AuthController.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+namespace QuizApp\Controllers;
+
+use Framework\Contracts\RendererInterface;
+use Framework\Controller\AbstractController;
+use Framework\Http\Request;
+use Framework\Http\Response;
+use Framework\Http\Stream;
+use QuizApp\Exceptions\UserNotFoundException;
+use QuizApp\Services\AuthServices;
+
+class AuthController extends AbstractController
+{
+    private $authServices;
+
+    /**
+     * UserController constructor.
+     * @param RendererInterface $renderer
+     * @param AuthServices $authServices
+     */
+    public function __construct(RendererInterface $renderer, AuthServices $authServices)
+    {
+        parent::__construct($renderer);
+        $this->authServices = $authServices;
+    }
+
+    public function showLogin()
+    {
```

Comment:
Error messages should be separated from the authService. 
You should have a place in ur app where to put/retrieve error(and not only error might be success just as well) messages. (A messageManager or smth)
In my opinion it should not be the responsibility of the auth service to keep track of the error messages you want to display in frontend.